src/styles/constants/index.scss

```
@@ -2,6 +2,7 @@
 @use './theme-light.scss';
 @use './theme-dark.scss';
 @use './theme-print.scss';
+@use './animations.scss';
```

Stylelint (17.11.1)

[error] Unexpected extension ".scss" in `@use` (scss/load-partial-extension)

(scss/load-partial-extension)